central/complianceoperator/v2/scanconfigurations/datastore/datastore_impl.go

```diff
@@ -125,7 +126,15 @@ func (ds *datastoreImpl) UpdateClusterStatus(ctx context.Context, scanID string,
 		return errors.Errorf("Unable to find scan configuration id %q", scanID)
 	}
 
+	// Need to build a deterministic ID from clusterID and scanID to ensure we always have the latest status
+	scanUUID, err := uuid.FromString(scanID)
```

Comment:
I don't know why, but it feels like it should be uuid from cluster and then the scan uuid is being generated from that?

Reply:
I basically flipped a coin on that.  I will flip it to cluster.